packages/twenty-front/src/modules/object-metadata/states/objectMetadataItemFamilySelectorById.ts

```diff
@@ -0,0 +1,26 @@
+import { selectorFamily } from 'recoil';
+
+import { objectMetadataItemsState } from '@/object-metadata/states/objectMetadataItemsState';
+import { ObjectMetadataItem } from '@/object-metadata/types/ObjectMetadataItem';
+
+type ObjectMetadataItemSelector = {
+  objectId: string;
+};
+
+export const objectMetadataItemFamilySelectorById = selectorFamily<
```

Comment:
is this one useful, I feel we are going to have too many selectors and it will complexify the devXP